cluster_tools/cluster_tools/remote.py

```diff
@@ -115,10 +115,7 @@ def setup_logging(
         )
         meta_data["logging_setup_fn"](log_file_path)
     else:
-        logging_config = meta_data.get(
-            "logging_config",
-            {"level": logging.DEBUG, "format": "%(asctime)s %(levelname)s %(message)s"},
-        )
+        logging_config = meta_data.get("logging_config", dict())
```

Comment:
Nitpick: Using an empty dict will also reset our default logging `format` to Python's default (`"%(levelname)s:%(name)s:%(message)s"` [Source Code](https://github.com/python/cpython/blob/1a84bdc2371ada60c01c72493caba62c9860007b/Lib/logging/__init__.py#L546)). I think that is acceptable.